src/routes/session_parser.ts

```diff
@@ -12,11 +12,11 @@ const sessionParser = session({
     cookie: {
         path: config.Session.cookiePath,
         httpOnly: true,
-        maxAge: 24 * 60 * 60 * 1000 // in milliseconds
+        maxAge:  config.Session.cookieMaxAge
```

Comment:
I believe we should change the unit of `cookieMaxAge` from milliseconds to seconds. Seems `Max-Age` cookie property is also expressed in seconds [[mdn]](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#max-agenumber).

I'm pretty sure sub-second precision is not very relevant for a cookie expiry date.

Reply:
I just used the same value that https://www.npmjs.com/package/express-session is expecting to receive.

I checked their code, and you can see why they expect ms:
https://github.com/expressjs/session/blob/4930de729107448e4b34b5bd39f138f71f1561f6/session/cookie.js#L93

I don't mind changing setting on our end to in seconds though :-)
We then just would need to do a simple `* 1000` before passing it over to the sessionParser here (or in the "/routes/login" login function - which is where the *actual* session expiration is set, as I found out earlier today – changed locally already, but didn't push yet).

Let me know what you think

Reply:
@pano9000 , multiplying by 1000 sounds good. `express-session` seems to do an interesting thing with the value, since `maxAge` does not set the `maxAge` setting, but calculates an `expires` which does indeed take milliseconds.

Based on what you mentioned, we also have to clarify that the `maxAge` is only applicable only when "remember me" is checked.

If we set it in `login`, maybe we shouldn't set it in the `session_parser` too, since it feels redundant. Up to you.